ansible/roles/eessi/templates/default.local.j2

```diff
@@ -0,0 +1,2 @@
+CVMFS_CLIENT_PROFILE="single"
```

Comment:
This looks a bit worrying: https://github.com/cvmfs/cvmfs/blob/10c74bfb3a383709ca2de3e1ed676e6801eafdcb/mount/default.conf#L25

It wouldn't be hard to add a squid proxy (we've already got a containerised role), can we ask what JG thinks.